Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -572,4 +582,11 @@ private int getConceptsCountForConceptList(String listName) {
         alternativeIdService.addAlternativeIds(conceptEntries);
         return conceptEntries;
     }
+
+    @Override
+    public void addWordnetConceptEntry(ConceptEntry entry, String userName)
```

Comment:
and similarly the method ```addWordNetConcepts``` in ```ConceptEditService``` needs to be named in a way that makes clear it's not adding a concept. It just updates the index.